students/humberto_gonzalez/session05/mailroom.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 13 17:53:34 2019
+
+@author: humberto gonzalez
+"""
+
+import sys  
+import tempfile
+import os
+
+
+donor_db = {"Tyrod Taylor": [1000.00, 45.50],
+            "Jarvis Landry": [150.25],
+            "Philip Rivers": [650.23, 40.87, 111.32],
+            "Melvin Gordon": [1677.25, 4300.23, 10532.00],
+            "Mike Williams": [230.56, 12.45, 11.00],
+            "Antonio Brown": [100.00, 88.88]
+            }
+
+donors = list(donor_db.keys())
+
+def getKey(item):
+    return item[1]
+
+def ordered_db(db):
+    keys = list(db.keys())
+    temp = []
+    for name in keys:
+        donation_total = sum(db[name])
+        temp.append((name,donation_total))
```

Comment:
simplify this:
```
def ordered_db():
    for donor, donations in donor_db.items():
        temp.append((donor, sum(donations)))
```


however, given that you should use comprehensions in this lesson:
```
name_and_donations_sum = [name, sum[donations] for name, donations in donor_db.items()]
```